src/compile/mark/mark.ts

```diff
@@ -174,12 +174,12 @@ function getStackGroups(model: UnitModel) {
       }
     }
 
+    const groupByField = model.fieldDef(model.stack.groupbyChannel);
     // For bin we have to add bin channels.
```

Comment:
Consider moving this comment to before the if statement, and describe timeUnit?

Reply:
Why moving?   Unlike Vega codebase, we prefer to declare things where they are used.

Reply:
(if you agree, please go ahead and merge. :D)

Reply:
I thought the comment is describing the what the if statement is doing?

Reply:
oh you mean this if below.